src/patterns/organisms/newsletter/newsletter.hbs

```diff
@@ -0,0 +1,86 @@
+notes: |
+  - Some newsletters are only available in one language, so don’t require a language selection.
+---
+
+<aside class="mzp-c-newsletter">
+  <div class="mzp-c-newsletter-image">
+    <img src="{{@root.baseurl}}/static/img/newsletter/newsletter-image.png" width="346" height="346" alt="">
+  </div>
+
+  <form id="newsletter-form" class="mzp-c-newsletter-form" name="newsletter-form" action="/path/to/handler/" method="post">
+    <header class="mzp-c-newsletter-header">
+      <h3 class="mzp-c-newsletter-title">Love the Web?</h3>
+      <p class="mzp-c-newsletter-tagline">Get the Mozilla newsletter and help us keep it open and free.</p>
+    </header>
+
+    <fieldset class="mzp-c-newsletter-content">
+      <div class="mzp-c-form-errors" id="newsletter-errors"></div>
+
+      <p>
+        <label for="email">Your email address</label>
+        <input type="email" id="email" name="email" placeholder="yourname@example.com" required aria-required="true">
+      </p>
+
+      <div id="newsletter-details" class="mzp-c-newsletter-details">
+        <p>
+          <label for="country">Country</label>
+          <select id="country" name="country" required aria-required="true">
+            {{#each (data "items")}}
+              <option value="{{country}}">{{country}}</option>
+            {{/each}}
+              <option>And so on...</option>
+          </select>
+        </p>
+
+        <p>
+          <label for="language">Language</label>
+          <select id="language" name="lang" required aria-required="true">
+            {{#each (data "items")}}
+              <option value="{{language}}">{{language}}</option>
+            {{/each}}
+          </select>
+        </p>
+
+        <fieldset class="mzp-u-inline">
+          <legend>Format</legend>
+          <p>
+            <label for="format-html" class="mzp-u-inline">
+              <input type="radio" id="format-html" name="fmt" value="H" checked> HTML
+            </label>
+            <label for="format-text" class="mzp-u-inline">
+              <input type="radio" id="format-text" name="fmt" value="T"> Text
+            </label>
+          </p>
+        </fieldset>
+
+        <p>
+          <label for="privacy" class="mzp-u-inline">
+            <input type="checkbox" id="privacy" name="privacy" required aria-required="true">
+            I’m okay with Mozilla handling my info as explained in <a href="#">this Privacy Notice</a>
+          </label>
+        </p>
+      </div>
+
+      <p class="mzp-c-form-submit">
+        <button type="submit" class="mzp-c-button" id="newsletter-submit">Sign up now</button>
+        <span class="mzp-c-fieldnote">We will only send you Mozilla-related information.</span>
+      </p>
+    </fieldset>
+  </form>
+
+  <div id="newsletter-thanks" class="mzp-c-newsletter-thanks">
```

Comment:
Is there any way to see these styles in the example without the form being functional?

Reply:
I've set up the demo page to show different states of the form (normal, errors, success).